health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/editVillagePopulationPopUP.js

```
@@ -27,34 +29,31 @@ const EditVillagePopulationPopUp = ({ onClose, census, onSuccess }) => {
   // Define the mutation configuration
   const mutation = Digit.Hooks.useCustomAPIMutationHook({
     url: "/census-service/_update", // Replace with the appropriate API endpoint
   });
 
   const handleSave = async () => {
-    // Determine workflow action based on user roles
-    let workflowAction = ""; // Default action
-
-    // Check user roles and set workflow action accordingly
+    let workflowAction = "";
     if (userRoles && userRoles.includes('POPULATION_DATA_APPROVER')) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)